CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -129,6 +148,8 @@ Describe "Single compute node protocol tests with utils" {
 
 
     BeforeAll {
+        Initialize-IisTcpTestDockerImage -Session $Session
```

Comment:
[Refinement] Maybe we should remove this docker image in AfterAll

Reply:
cc @jacek-izykowski 

I strongy disagree with removing the image in `AfterAll` as it prevents docker from caching the image. This slows down both manual testing and testing in CI (when there'll be multiple tests using this image).

Without the removal, the tests are still independent, the only side effect is that `Initialize-DockerImage` will sometimes be faster.